app/src/main/java/com/example/calculatorhw/NotesAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.calculatorhw
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.calculatorhw.databinding.ItemNoteBinding
+import java.util.ArrayList
+
+class NotesAdapter(private val items: MutableList<String>, private val itemClick: (String) -> Unit) :
```

Comment:
не нужно завязываться на конкретных списках, плюс потенциально если держать ссылку на этот список снаружи адаптера, то можно поменять лист, эти изменения также повлияют на адаптер
```suggestion
class NotesAdapter(private val items: List<String>, private val itemClick: (String) -> Unit) :
```